home.html

```diff
@@ -0,0 +1,127 @@
+             </p>
+             <p class="time">2021.10.15(Thu) ~ 16(fri)</p>
+             <p class="venue">@ National Stadium of Abuja, Art Center Lagos and more</p>
+          </section>
+          <section class="program-section">
+              <h2 class="program-header">Main program</h2>
+              <ul class="flex program-ul">
+                  <li class="program-list">
+                      <ul class="flex program-flex">
+                          <li><i class="fa fa-volume-up program-icon"></i></li>
+                          <li class="program-info">Lecture</li>
+                          <li class="program-talk">Listen to the speaker from various countries about the fast growing demand of web designer and developers.</li>
+                      </ul>
+                  </li>
+                  <li class="program-list">
+                      <ul class="flex program-flex">
+                          <li><i class="fa fa-home program-icon"></i></li>
+                          <li class="program-info">Forum</li>
+                          
+<li class="program-talk">Appreciate use of web accessibilty, web designing and app creations from art center, Lagos.</li>
+                      </ul>
+                  </li>
+                  <li class="program-list">
+                    <ul class="flex program-flex">
+                        <li><i class="fa fa-volume-up program-icon"></i></li>
+                        <li class="program-info">Lecture</li>
+                        <li class="program-talk">Listen to the speaker from various countries about the fast growing demand of web designer and developers.</li>
+                    </ul>
+                </li>
+                <li class="program-list">
+                    <ul class="flex program-flex">
+                        <li><i class="fa fa-home program-icon"></i></li>
+                        <li class="program-info">Forum</li>
+                        
+<li class="program-talk">Appreciate use of web accessibilty, web designing and app creations from art center, Lagos.</li>
+                    </ul>
+                </li>
+                <li class="program-list">
+                    <ul class="flex program-flex">
+                        <li><i class="fa fa-volume-up program-icon"></i></li>
+                        <li class="program-info">Lecture</li>
+                        <li class="program-talk">Listen to the speaker from various countries about the fast growing demand of web designer and developers.</li>
+                    </ul>
+                </li>                 
+              </ul>
+               <a href="#" class="program-anchor">join Global Tech 2021</a>
+          </section>
+          <section class="featured-section">
+              <h2 class="featured-header">Featured Speakers</h2>
+              <ul class="flex featured-ul">
+              </ul>
+          </section>
```

Comment:
- [ ] OPTIONAL: This section has issues related to maintaining the original design. Particularly the spacing is visibly away from the original design. Please see the attached screenshot and take this as an optional comment to work on in your spare time. 
![image](https://user-images.githubusercontent.com/64894927/124193885-4922c400-dad0-11eb-9b9c-6af7ce37132a.png)